.github/ct-install.yaml

```diff
@@ -24,6 +24,7 @@ excluded-charts:
   - charts/stable/discordgsm
   - charts/stable/facebox
   - charts/stable/foundryvtt
+  - charts/stable/local-ai
```

Comment:
What blocks it from starting on CI?

Reply:
between my commits it switches from this:
```
Error from server (BadRequest): container "local-ai-6fz5cneyse" in pod "local-ai-6fz5cneyse-554c4ffb99-ddrck" is waiting to start: ContainerCreating
Error printing details: failed waiting for process: exit status 1
```

to:

```
failed to try resolving symlinks in path "/var/log/pods/local-ai-qbb4dfigli_local-ai-qbb4dfigli-7574f4cbb7-8gwzn_7d50172b-baae-4cf3-b6e7-c55319daa1b6/local-ai-qbb4dfigli/0.log": lstat /var/log/pods/local-ai-qbb4dfigli_local-ai-qbb4dfigli-7574f4cbb7-8gwzn_7d50172b-baae-4cf3-b6e7-c55319daa1b6: no such file or directory
```

i dont get these errors when i run this locally